addons/velero/1.6.0/Manifest

```diff
@@ -3,5 +3,7 @@ image restic-restore velero/velero-restic-restore-helper:v1.6.0
 image velero-aws velero/velero-plugin-for-aws:v1.2.0
 image velero-gcp velero/velero-plugin-for-gcp:v1.2.0
 image velero-azure velero/velero-plugin-for-microsoft-azure:v1.2.0
+image local-volume-provider replicated/local-volume-provider:v0.1.0
```

Comment:
will this plugin not work with velero 1.5.x? i don't see it included there

Reply:
I did some compatibility testing and was able to successfully do a backup and restore on 1.5.x with LVP, so I will add this.